src/test/resources/cucumber.properties

```diff
@@ -0,0 +1 @@
+cucumber.publish.enabled=true
```

Comment:
Nice 👍🏽 This will create and deploy the report. If you want to, you can copy that link in the PR summary!